plugins/GSdx/GS.cpp

```diff
@@ -947,6 +947,15 @@ EXPORT_C GSgetTitleInfo2(char* dest, size_t length)
 	strcpy(dest, s.c_str());
 }
 
+EXPORT_C GSgetImageSize(int *image_width = 0, int *image_height = 0)
```

Comment:
Besides I'm not sure they do what we expect. For me it sets them to null pointer (but I'm not 100% sure)